_includes/footer.njk

```diff
@@ -0,0 +1,13 @@
+<footer>
+    <div class="container">
+      <img src="/img/rf-logo-transparent.png" alt="rust foundation logo"/>
+      <div>
+      {% for policy in collections.policies %}
+        <a href="/policies/{{ policy.data.title | replace(" ", "-") }}">{{ policy.data.title }}</a>
```

Comment:
It looks like this isn't quite generating the link right - locally I get a 404 because http://localhost:8080/policies/anti-trust-policy/ is the right URL, but this generates http://localhost:8080/policies/Anti-Trust-Policy

Reply:
the links are case-sensitive and the trailing slash doesn't matter- i can add code to add the slash and lowercase it but i'm not convinced it's worth it- is there a reason you call it out?

Reply:
ok i added a change to lowercase the names but the trailing slash thing is a eleventy thing and i'd rather leave it as is